hack/compute/role.yaml

```diff
@@ -58,3 +58,13 @@ rules:
   - update
   - watch
   - patch
+- apiGroups:
+  - workload.kcp.dev
+  resources:
+  - synctargets
```

Comment:
Are we sure we need to add this permission? Isn't this covered through the APIExport permission claim?